test/sql/CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ if (CMAKE_BUILD_TYPE MATCHES Debug)
   endif (USE_OPENSSL)
 endif (CMAKE_BUILD_TYPE MATCHES Debug)
 
+if (${PG_VERSION_MAJOR} EQUAL "10")
+  list(APPEND TEST_FILES
+    rowsecurity-10.sql)
```

Comment:
Seeing this made me realize that `generated_as_identity.sql` got dropped from this file in a rebase/merge during the background worker stuff. Can you add it back in from major version > 9? The files are there, its just never added to the list.